src/main/java/blackjack/user/Participants.java

```diff
@@ -0,0 +1,84 @@
+package blackjack.user;
+
+import blackjack.card.CardDeck;
+import java.util.Collections;
+import java.util.List;
+
+public class Participants {
+
+    private static final int PARTICIPANTS_MIN_SIZE = 1;
+    private static final int PARTICIPANTS_MAX_SIZE = 25;
+    private static final int INITIAL_DISTRIBUTE_CARD_NUMBER = 2;
+    private static final int EXTRA_DISTRIBUTE_CARD_NUMBER = 1;
+
+    private final Dealer dealer;
+    private final List<Player> players;
+
+    public Participants(final Dealer dealer, final List<Player> players) {
+        validatePlayerNumber(players);
+        validateDuplicatePlayerNames(players);
+
+        this.dealer = dealer;
+        this.players = players;
+    }
+
+    private void validatePlayerNumber(final List<Player> players) {
+        if (players.size() < PARTICIPANTS_MIN_SIZE) {
```

Comment:
```suggestion
        if (players.isEmpty()) {
```
List가 비어있는지 확인하면 되겠네요! 그러면 PARTICIPANTS_MIN_SIZE 상수도 불필요해 보여요~

Reply:
인원수의 최소값이 변경될 수 있을 것 같아서 상수로 빼두었는데 지금 상황에서는 비어있는지 확인해도 되겠군요!